src/main/java/uk/gov/hmcts/reform/bsp/services/DailyChecksService.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.bsp.services;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bsp.integrations.SlackClient;
+
+import java.util.List;
+
+@Service
+@Slf4j
```

Comment:
we aren't logging anything so don't need, happy to keep if you think logs will be added as part of ticket